client/state/admin-menu/fallback-data.json

```diff
@@ -0,0 +1,479 @@
+[
+    {
+        "icon": "dashicons-feedback",
+        "slug": "feedback",
+        "title": "Feedback",
```

Comment:
Is there a way to make these titles translatable?

Reply:
The only way would be to implement the fallback as a JavaScript object rather than pure JSON. We could then use the `translate` util to translate all the titles. That's totally doable though. Great spot!

Reply:
Also...are we going into too much detail as this stage?